scripts/fedora/openstack.sh

```diff
@@ -9,7 +9,6 @@ dracut -f
 
 if [ $(uname -m)=="ppc64" -o $(uname -m)=="ppc64le" ]
   then
-    yum -y update
```

Comment:
Why remove this ? Also, would this affect fedora since it's `yum` and not `dnf` ?